src/main/java/ch/njol/skript/effects/EffToggle.java

```diff
@@ -40,63 +40,80 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
-@SuppressWarnings("deprecation")
 @Name("Toggle")
-@Description("Toggle the state of a block.")
+@Description("Toggle a boolean or the state of a block.")
 @Examples({"# use arrows to toggle switches, doors, etc.",
 		"on projectile hit:",
 		"\tprojectile is arrow",
-		"\ttoggle the block at the arrow"})
-@Since("1.4")
+		"\ttoggle the block at the arrow",
+		"",
+		"# With booleans",
+		"toggle gravity of player"
+})
+@Since("1.4, INSERT VERSION (booleans)")
 public class EffToggle extends Effect {
 	
 	static {
-		Skript.registerEffect(EffToggle.class, "(close|turn off|de[-]activate) %blocks%", "(toggle|switch) [[the] state of] %blocks%", "(open|turn on|activate) %blocks%");
+		Skript.registerEffect(EffToggle.class, 
+				"(close|turn off|de[-]activate) %blocks%",
+		 		"(toggle|switch) [[the] state of] %booleans/blocks%",
+		  		"(open|turn on|activate) %blocks%");
```

Comment:
Coming a bit late with this suggestion, but wouldn't 'enable' and 'disable' be nice alternatives? Not a big deal though.

Reply:
Not really, because this effect *toggles* the boolean, changing the value from true to false, and false to true. Using "enable" or "disable" instead would make the value "true" only or "false" only. A simple "set" would be enough in that case.